website/content/docs/k8s/multiport/configure.mdx

```diff
@@ -59,22 +64,28 @@ Then install Consul to your Kubernetes cluster using either the `consul-k8s` CLI
 
 <Tab heading="consul-k8s CLI" group="consul-k8s">
 
-For platforms other than Mac OSX amd64, refer to [Install a previous version](/consul/docs/k8s/installation/install-cli#install-a-previous-version) for instructions on how to install a specific version of the `consul-k8s` CLI prior to running `consul-k8s install`. 
-
 ```shell-session
-$ export VERSION=1.3.0-rc1 && \
-    curl --location "https://releases.hashicorp.com/consul-k8s/${VERSION}/consul-k8s_${VERSION}_darwin_amd64.zip" --output consul-k8s-cli.zip
-$ unzip -o consul-k8s-cli.zip -d ~/consul-k8s
-$ export PATH=$PATH:$HOME/consul-k8s
 $ consul-k8s install -config-file=values.yaml
 ```
 
+To download a compatible version of `consul-k8s` and add it to your `PATH`, run the following commands instead:
+
+```shell-session
+$ export VERSION=1.3.0 && \
```

Comment:
Drive by comment, in this case one could just install the latest `consul-k8s` so no need to install via these explicit version instructions

Reply:
I'd replace with

```shell-session
$ brew install hashicorp/tap/consul-k8s
$ consul-k8s install -config-file=values.yaml
```